packages/twenty-front/src/modules/object-metadata/utils/formatFieldMetadataItemAsColumnDefinition.ts

```diff
@@ -28,6 +34,8 @@ export const formatFieldMetadataItemAsColumnDefinition = ({
     position,
     fieldMetadataId: field.id,
     label: field.label,
+    showLabel,
```

Comment:
I have added these on FieldDefinition so they can be passed through the FieldContext provider (this avoid having to drill down props and let the page / section decide if we should display label or not)